CHANGELOG.md

```diff
@@ -1,37 +1,42 @@
+* `azuredevops_branch_policy_build_validation`  - add `filename_patterns` support for repository build policy [#62](https://github.com/microsoft/terraform-provider-azuredevops/issues/62)
+* `azuredevops_git_repository`
+    - Use `default_branch` as the name of an initialized branch [#89](https://github.com/microsoft/terraform-provider-azuredevops/issues/89)
     - Add support for import Git repository [#45](https://github.com/microsoft/terraform-provider-azuredevops/issues/45)
-* **`azuredevops_build_definition`:**
-    - Add Support for GitHub enterprise as a build definition repository type [#97](https://github.com/microsoft/terraform-provider-azuredevops/pull/97)
+* `azuredevops_build_definition`
+    - Add Support for GitHub enterprise as a build definition repository type [#97](https://github.com/microsoft/terraform-provider-azuredevops/issues/97)
     - Add Support for report build status configuration [#63](https://github.com/microsoft/terraform-provider-azuredevops/issues/63)
 
 BUG FIX:
-* **`azuredevops_serviceendpoint_github`: GitHub service connection API breaking change** [#72](https://github.com/microsoft/terraform-provider-azuredevops/issues/72)
-* **All service connection resources: Terraform crashes when the service connection description is set to an empty string** [#60](https://github.com/microsoft/terraform-provider-azuredevops/pull/60)
-* **`azuredevops_build_definition`: Build Definition creation failed when repository type is GitHub** [#65](https://github.com/microsoft/terraform-provider-azuredevops/issues/65)
+* All service connection resources - Terraform crashes when the service connection description is set to an empty string [#60](https://github.com/microsoft/terraform-provider-azuredevops/issues/60)
+* Resource import - set the project ID to `project_id` [#172](https://github.com/microsoft/terraform-provider-azuredevops/issues/172)
+* `azuredevops_build_definition` - build Definition creation failed when repository type is GitHub [#65](https://github.com/microsoft/terraform-provider-azuredevops/issues/65)
+* `azuredevops_serviceendpoint_github` - GitHub service connection API breaking change [#72](https://github.com/microsoft/terraform-provider-azuredevops/issues/72)
 
 BREAKING CHANGES:
-* `azuredevops_git_repository` - `initialization` now is a required configuration.  [#54](https://github.com/microsoft/terraform-provider-azuredevops/issues/54)
+* Data source: `azuredevops_project` - remove `project_name`, add `project_identifier` as the required configuration [#176](https://github.com/microsoft/terraform-provider-azuredevops/issues/176)
+* `azuredevops_git_repository` - `initialization` is a required configuration [#54](https://github.com/microsoft/terraform-provider-azuredevops/issues/54)
+* `azuredevops_project` - rename `project_name` to `name` [#179](https://github.com/microsoft/terraform-provider-azuredevops/issues/179)
 
 ## 0.0.1 (June 18, 2020)
 
```

Comment:
While this update is fine, there's an issue with the previous links pointing to wrong issues example `* **New Resource** `azuredevops_variable_group` [#21](https://github.com/microsoft/terraform-provider-azuredevops/issues/21)`

Is there a ticket for this?

Reply:
There's no ticket for this. The issue links  for v0.0.1 is linked to the old ADO repo which we have deleted. I removed the broken links.